### PR TITLE
Feature/image modal

### PR DESCRIPTION
**Pull request by nidomiro**
This PR adds support for clickable images via [magnific-popup](https://dimsemenov.com/plugins/magnific-popup/).

It works for AsciiDoc but Markdown has to be tested.

Usage in AsciiDoc:
```asciidoc
[#img-config-change]
.What happens on config change
image::git-config-change.svg[alt="Git config change sequence", link="git-config-change.svg"]
```
Result: https://nidomiro.de/article/server-config-from-git/#img-config-change

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -69,8 +69,8 @@
     <meta itemprop="description" content="{{ .Scratch.Get "description" }}">
     <meta property="og:title" content="{{ .Scratch.Get "title" }}">
     <meta property="og:description" content="{{ .Scratch.Get "description" }}">
-    <meta property="og:image" content="{{ .Scratch.Get "image" | absURL }}">
-    <meta property="og:url" content="{{ .Permalink | absURL }}">
+    <meta property="og:image" content="{{ .Scratch.Get "image" | relURL }}">
```
Comment (Lednerb) [inline layouts/_default/baseof.html:72]: There was a bug with having a relative URL in this place (https://github.com/Lednerb/bilberry-hugo-theme/issues/168)

Maybe we should use always use absolute URLs in the `meta` tags?

File: testing/testing-example-site/config.toml [hunk 2]
```diff
@@ -2,6 +2,10 @@ title = "My cool new Blog"
 baseURL = "https://example.com/"
 theme = "bilberry-hugo-theme"
 
+# Force absolute urls
+relativeURLs = false
+canonifyURLs = true
+
```
Comment (Lednerb) [inline testing/testing-example-site/config.toml:8]: This option for having absolute URLs should also be added to the docs.

File: layouts/_default/baseof.html (hunk 1 above)
Comment (nidomiro) [reply layouts/_default/baseof.html:72]: The reason I changed everything to relative is that I want to be able to access the dev-server from my smartphone and test the site there. I don't think making these meta urls absolute will prohibit hat.
I'll test it ;)

File: testing/testing-example-site/config.toml (hunk 2 above)
Comment (nidomiro) [reply testing/testing-example-site/config.toml:8]: It's now documented: https://github.com/nidomiro/bilberry-hugo-theme/tree/feature/image-modal#absolute-vs-relative-urls
The usage of the image-Modal is documented here: https://github.com/nidomiro/bilberry-hugo-theme/tree/feature/image-modal#absolute-vs-relative-urls

**Comment by nidomiro** [comment]
Please do not merge it right now.
I just found a bug that causes a wrong image modal if an anchor is in a paragraph without an image.

**Comment by nidomiro** [comment]
Sadly you have to use html in markdown to use this functionallity.
Asciidoc works out of the box.

It's ready to merge from my point of view.

**Comment by nidomiro** [comment]
@Lednerb whats your status on this one?

**Comment by Lednerb** [comment]
This feature will be merged soon.

I have to refactor some things first and will check the changes intruduced by this PR on my local testing environment.

**Comment by Lednerb** [comment]
Closing here in favor of #243 and #244

**Pull request closed without merging**